index.html

```diff
@@ -33,19 +37,95 @@ <h1 class="logo">
         hesitate and contact me.
       </p>
 
-      <button type="button">Lets Connect</button>
+      <button type="button" class="connect">Lets Connect</button>
       <img class="design-one" src="bgbg.png" alt="hardwork bro" />
```

Comment:
- [ ] Kindly make this a background image and not an actual image. When images are used as part of the design of a web page and not actual data, they should be implemented as background images. Kindly fix this. 👍